jekyll/_cci2/create-an-orb.adoc

```diff
@@ -90,31 +87,31 @@ The `circleci orb init` command is called, followed by a path that will be creat
       No, I will handle everything myself.
 ```
 
-When choosing the fully automated option, the link:https://github.com/CircleCI-Public/Orb-Template[Orb-Template] will be downloaded and automatically modified with your customized settings. The project will be followed on CircleCI with an automated CI/CD pipeline included.
+When choosing the fully automated option, the link:https://github.com/CircleCI-Public/Orb-Template[Orb-Template] is downloaded and automatically modified with your customized settings. The project will be followed on CircleCI with an automated CI/CD pipeline included.
 
-For more information on the included CI/CD pipeline, see the [Orb Publishing Process]({{site.baseurl}}/creating-orbs/) documentation.
+For more information on the included CI/CD pipeline, see the link:/docs/creating-orbs/[orb publishing process] page.
 
-Alternatively, if you would simply like a convenient way of downloading the link:https://github.com/CircleCI-Public/Orb-Template[Orb-Template] you can opt to handle everything yourself. When choosing the manual option, see link:/docs/orb-author-validate-publish/[Manual Orb Authoring Process] for instructions on how to author and publish your orb.
+NOTE: If you would simply like a convenient way of downloading the link:https://github.com/CircleCI-Public/Orb-Template[Orb-Template] you can opt to handle everything yourself. When choosing the manual option, see link:/docs/orb-author-validate-publish/[manual orb authoring process] for instructions on how to author and publish your orb.
 
 === 4. Follow the prompts to set up your orb
 
 In the background, the `orb init` command copies and customizes the link:https://github.com/CircleCI-Public/Orb-Template[orb template] based on your inputs. There are detailed `README.md` files within each directory that contain helpful information specific to the contents of each directory. You will also be asked for the remote git repository URL that you obtained back in step 1.
 
-The link:https://github.com/CircleCI-Public/Orb-Template[Orb Template] contains a full CI/CD pipeline (described in link:/docs/creating-orbs/[Orb Publishing Process]) which automatically link:/docs/orb-concepts/#orb-packing[packs], link:/docs/testing-orbs/[tests], and link:https://circleci.com/docs/creating-orbs/[publishes] your orb.
+The link:https://github.com/CircleCI-Public/Orb-Template[orb template] contains a full CI/CD pipeline (described in link:/docs/creating-orbs/[orb publishing process]), which automatically link:/docs/orb-concepts/#orb-packing[packs], link:/docs/testing-orbs/[tests], and link:https://circleci.com/docs/creating-orbs/[publishes] your orb.
```

Comment:
The page still has inconsistent capitalization of Orb Template vs orb template (this is not the only location, but one where both are visible next to each other)
![Screen Shot 2022-12-15 at 10 37 42 AM](https://user-images.githubusercontent.com/20114388/207902864-5c83922d-0a64-41a8-a06b-e9c605b36d97.png)